docker-image/v0.12/alpine-logentries/Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN set -ex \
     && gem install fluent-plugin-secure-forward \
     && gem install fluent-plugin-record-reformer \
     && gem install fluent-plugin-prometheus -v 0.4.0 \
+    && gem install fluent-plugin-splunk-hec \
```

Comment:
Shouldn't we specify a version for these?

Reply:
Updated with specific version